packages/base-styles/_variables.scss

```diff
@@ -76,6 +76,8 @@ $block-selected-to-content: $block-edge-to-content - $block-selected-margin - $b
 $block-selected-child-to-content: $block-selected-to-content - $block-selected-child-margin - $block-selected-child-border-width;
 $block-custom-appender-to-content: $block-selected-margin - $block-selected-border-width;
 $block-media-container-to-content: $block-selected-child-margin + $block-selected-border-width;
+$block-selected-vertical-margin-descendant: 3 * $block-selected-child-to-content;
+$block-selected-vertical-margin-child: 3 * $block-selected-child-to-content - 6 * $block-selected-child-border-width;
```

Comment:
I don't understand these calculations, can you explain what's going on here?

Reply:
I have answer to that after pushing changes because this one is out of date. I did a mistake implementing the logic here.